src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/TestFixtures.kt

```diff
@@ -71,7 +71,11 @@ fun recallWithPopulatedFields(
   reasonsForRecallOtherDetail = randomString(),
   agreeWithRecall = AgreeWithRecall.values().random(),
   agreeWithRecallDetail = randomString(),
-  currentPrison = randomString()
+  currentPrison = randomString(),
+  additionalLicenceConditions = true,
```

Comment:
(I thought we had boolean properties before this one but ...).  The pattern in this test for random valid data to be used.  Should we not stick with that pattern in these new properties?